ui/package.json

```diff
@@ -193,7 +193,7 @@
   "resolutions": {
     "cryptiles": "^4.1.2",
     "eslint-utils": "^1.4.1",
-    "ember-basic-dropdown": "3.0.19",
+    "ember-basic-dropdown": "4.0.5",
```

Comment:
Do we need this anymore? I can't remember for sure but I recall that this was necessary for `ember-basic-dropdown-hover` which has been removed on the 3.28 branch. Would these updates be more relevant off of that branch rather than main?

Reply:
Interesting. Let me run these updates on the 3.28 branch and see if i hit any conflicts.